samples/nfc/tag_reader/src/main.c

```diff
@@ -0,0 +1,192 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include <errno.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <drivers/st25r3911b/nfca.h>
+
+#define NFC_T2T_READ_CMD 0x30
+#define NFC_BLOCK_TO_READ_NUM 0
+
+#define NFC_T2T_READ_CMD_LEN 0x02
+#define NFC_T2T_BLOCK_LEN 0x10
+
+#define NFC_TX_DATA_LEN NFC_T2T_READ_CMD_LEN
+#define NFC_RX_DATA_LEN NFC_T2T_BLOCK_LEN
+
+#define TRANSMIT_DELAY 3000
+#define ALL_REQ_DELAY 2000
+#define T2T_READ_CMD_FDT 2068
+
+u8_t tx_data[NFC_TX_DATA_LEN] = {NFC_T2T_READ_CMD, NFC_BLOCK_TO_READ_NUM};
```

Comment:
static const

Reply:
`const` is not possible spi buffer discard it.